content/physical_properties/electrical_conductivity/index.rst

```diff
@@ -3,81 +3,136 @@
 
-.. note::
 
-    Chargeability is often listed as an independent physical property
-    (such as density, magnetic susceptibility) and we will continue with that
-    in this EM resource. It is however, just an element that is necessary in
-    order to provide a complete description of electrical conductivity.
+.. figure:: ./images/Cole_Cole_example.png
+   :align: center
+   :figwidth: 80%
+   :name: fig_Cole_Cole_example
 
-Applicability
--------------
+   Cole-Cole conductivity (left) and resistivity (right) for :math:`\sigma_\infty` = 2 S/m, :math:`\rho_0` = 1 :math:`\Omega m`, :math:`\eta` = 0.5, :math:`\tau` = :math:`10^{-4}` s and :math:`C` = 0.5.
 
-.. todo::
+Relevance to Geophysical Applications
+-------------------------------------
 
-    Add thumbnail figure
+**Electrical Conductivity**
 
-Where is it diagnostic?
------------------------
+The majority of EM surveys exploit contrasts in electrical conductivity to image the subsurface. During :ref:`direct current resistivity <dcr_index>` (DCR) surveys for example, electrical current is forced through the Earth. The path taken by the current, as well as the measured data, depend on the subsurface conductivity distribution.
 
-.. todo::
+Many EM systems operate on the principles of EM induction. These include: :ref:`frequency-domain EM <airborne_fdem_index>` (FDEM), :ref:`time-domain EM <airborne_tdem_index>` (TDEM), :ref:`marine controlled source EM <marine_csem_index>` (CSEM) and :ref:`unexploded ordnance <uxo_index>` (UXO) surveys. During these surveys, a transmitter sends time-varying EM signals into the ground which subsequently induce electric currents. The strength of the induced currents and the secondary fields they produce are dependent on the distribution of subsurface conductivities.
 
-    Add thumbnail figure
+Data collected during :ref:`magnetotelluric <mt_index>` (MT) and :ref:`Z-axis Tipper EM <ztem_index>` (ZTEM) surveys also depend on the conductivity of the Earth. These methods rely on natural sources to generate EM responses. For MT, the relationships between measured components of the electric and magnetic fields provide insight regarding the Earth's electromagnetic impedance, and indirectly its electrical condutivity.
 
+**Chargeability**
+
+In comparison to most other rock types, sulphide-bearing rocks are highly chargeable; one exception being rocks with a high clay content. In sufficient abundances, sulphide-bearing rocks can have significant economic value. Unlike :ref:`DCR surveys <dcr_index>`, :ref:`IP surveys <ip_index>` can be used to distinguish chargeable and non-chargeable bodies, even if both are similarly conductive. :ref:`TDEM systems <airborne_tdem_index>` can also be used to recognize the presence of chargeable bodies, as they produce distinct time-domain responses. As a result, chargeability has become a unique diagnostic physical property used to located sulphide-bearing ore deposits.
 
 Contents
```

Comment:
I tried that at first. If I don't do it this way, the "relevance to geophysical applications" becomes a drop-down menu. I don't really like that. We could change contents to something else. Or we could consider some option with regards to altering the toctree setup.